src/protocols/ssh/settings.c

```diff
@@ -165,6 +166,12 @@ enum SSH_ARGS_IDX {
      */
     IDX_READ_ONLY,
 
+    /**
+     * Number of seconds between sending alive packets.  A default of 0
```

Comment:
I beefed up that comment to document that - I'll also verify what I put in the manual PR to make sure that correctly documents it.  I also changed some of the checks within the code to > 1 instead of > 0.  However, I'm not sure if it makes sense to actually change those and do any additional checks, or just let libssh2 handle it since libssh2 already handles it - as long as it's documented??  After changing those checks to > 1 it seems like maybe that would be more troublesome to someone trying to enable keepalives than just checking for > 0 and making sure documentations specifies that the minimum will never be < 2 seconds.  Or is there another check/warning/error I should implement here to throw something into the log file that warns of this configuration issue?